php/src/Docker/DockerActionManager.php

```diff
@@ -167,7 +167,7 @@ public function StartContainer(Container $container) : void {
         try {
             $this->guzzleClient->post($url);
         } catch (RequestException $e) {
-            throw new \Exception("Could not start container " . $container->GetIdentifier() . ": " . $e->getMessage());
+            throw new \Exception("Could not start container " . $container->GetIdentifier() . ": " . $e->getResponse()->getBody()->getContents());
```

Comment:
@surapuramakhil I finally found why the message is truncated: aparently getmessage is always truncated by the guzzle client but you can work around by using the above weird syntax. Wow, this was too complicate to find out.